### ISSUE TITLE
Decoder skip, add , concatenate

### ISSUE THREAD

**Issue by Songminkee** [nianticlabs/monodepth2#51]
in depth_decoder.py

def forward(self, input_features):
        self.outputs = {}

        # decoder
        x = input_features[-1]
        for i in range(4, -1, -1):
            x = self.convs[("upconv", i, 0)](x)
            x = [upsample(x)]
            if self.use_skips and i > 0:
                x += [input_features[i - 1]]
            x = torch.cat(x, 1)
            x = self.convs[("upconv", i, 1)](x)
            if i in self.scales:
                self.outputs[("disp", i)] = self.sigmoid(self.convs[("dispconv", i)](x))

        return self.outputs

1. x += [input_features[i - 1]]
In previous monodepth 1, concat was done.
As far as I known, it is generally use concat.
but now your code use add.
is there a reason for changing to 'Add function'?

2. x = torch.cat(x, 1)
I do not think duplicate information is needed, is there any significance in concat x?

**Comment by Songminkee** [comment]
Sorry, I misunderstand this code.
